server/server.js

```diff
@@ -79,6 +79,7 @@ app.options('*', corsMiddleware);
 app.use(Express.static(path.resolve(__dirname, '../dist/static'), {
   maxAge: process.env.STATIC_MAX_AGE || (process.env.NODE_ENV === 'production' ? '1d' : '0')
 }));
+app.use('/translations', Express.static('translations/locales/'));
```

Comment:
Rather than serve the translations statically, would it instead be better to bundle them in with the webpack build? Unfortunately, serving the files this way breaks the application in production, because they are not copied in the [Docker](https://github.com/processing/p5.js-web-editor/blob/develop/Dockerfile) build. On the staging server, fetching the translations is returning a 404 error.

cc @andrewn

Reply:
Or, the Dockerfile needs to be updated to copy the translations.

Reply:
Or or, the [webpack prod configuration](https://github.com/processing/p5.js-web-editor/blob/develop/webpack/config.prod.js) needs to be updated so that it copies the files into `dist/static`

Reply:
Ah, I didn’t realise that, sorry.

The translations shouldn’t be in the bundle as we don’t want users to have to download languages they won’t use. But I guess copying the translations into `dist/static` would be simplest.

Reply:
No worries at all! Let me know if y'all need any help with this.